README.md

```diff
@@ -110,41 +110,74 @@ This sample shows how to create custom AnalyzeRules for use within the Analyze w
-- An Addressable Group is assigned to an asset pack. Multiple groups can be assigned to the same asset pack. Just be mindful of the [size restrictions per delivery mode](https://developer.android.com/guide/playcore/asset-delivery#custom-asset-packs). Note, that at build time the AddressablesPlayerBuildProcessor will include all local content to the streaming assets path. This means that any local content will be automatically included in the streaming assets pack even if they are not assigned to a custom asset pack.
-  - Each group uses local build & load paths. At runtime these paths are overwritten to use asset pack locations.
-  - Also each group uses a custom AssetBundleProvider that ensures the asset pack containing the AssetBundle is installed/downloaded before attempting to load the bundle.
-- All content marked for "install-time" delivery is assigned to the streaming assets pack. Custom asset packs are created for all other delivery modes.
-- Use the custom build script to prepare bundled content for [custom asset pack creation](https://docs.unity3d.com/Manual/play-asset-delivery.html#custom-asset-packs). This will also create a json file that stores custom asset pack information to be used at runtime.
-- At runtime make sure that [generated asset packs](https://docs.unity3d.com/Manual/play-asset-delivery.html#generated-asset-packs) are downloaded, configure custom Addressables properties, and load all custom asset pack information.
-
-Setup Instructions:
-1. Configure Unity to build Android App Bundles and split the application binary. For more information see ["Using Play Asset Delivery"](https://docs.unity3d.com/Manual/play-asset-delivery.html#using-play-asset-delivery).
-   1. In File > Build Settings:
-      - Set the Platform to Android.
-      - If "Export Project is enabled", enable "Export for App Bundle". Otherwise, enable "Build App Bundle (Google Play)".
+An example project that shows how to use [Unity's Play Asset Delivery API](https://docs.unity3d.com/Manual/play-asset-delivery.html) with Addressables. SampleScene contains 3 buttons that will load or unload an asset that was assigned to an asset pack of a specific delivery type.
+
+ **Note**: [Play Asset Delivery](https://docs.unity3d.com/Manual/play-asset-delivery.html) requires Unity 2019.4+
+
+If you want to make [custom asset packs](https://docs.unity3d.com/Manual/play-asset-delivery.html#custom-asset-packs) use this project as a guide. Otherwise you don't need to change how Addressables is configured in your project. Just follow the instructions specified in the [Play Asset Delivery](https://docs.unity3d.com/Manual/play-asset-delivery.html) manual page.
+
+This is possible because the AddressablesPlayerBuildProcessor moves all content located in Addressables.BuildPath to the streaming assets path. Unity assigns streaming assets to the [generated asset packs](https://docs.unity3d.com/Manual/play-asset-delivery.html#generated-asset-packs).
+
+##### Basic Workflow
```

Comment:
Do these steps need to be performed in a specific order? If so, they should be in a numbered list. Otherwise, the bullet list you already have is good.